src/js/states/profile/edit/edit.module.js

```diff
@@ -51,7 +51,7 @@
 							controller: function( $scope, isLeader ) {
 								$scope.isLeader = isLeader;
 							},
-							template: '<button type="button" class="btn btn-sm btn-danger" ng-if="isLeader" ui-sref="admin">Administrator</button>'
+							template: '<button type="button" class="btn btn-sm btn-danger" ng-if="isLeader" ui-sref="admin" translate>Administrator</button>'
```

Comment:
The gulp gettext plugin probably isn't catching it in the template string. May need to move it to an html file or add a gettext noop to the controller to inform the plugin it exists.
https://angular-gettext.rocketeer.be/dev-guide/annotate-js/